cmd/abn.go

```diff
@@ -29,8 +29,3 @@ func newAbnCmd() *cobra.Command {
 	}
 	return cmd
 }
-
-// initialize with assert
-func init() {
-	rootCmd.AddCommand(newAbnCmd())
-}
```

Comment:
Please explain this change. Why?

Reply:
The order of `init()` executions is arbitrary in a golang program. It turns out that some of the init funcs for child flags (especially, the init func for the child flag `krun`) was assuming stuff that was supposed to happen in the init func of the parent flags (`k`).

I was able to catch this by **not** ignoring errors, and reporting them, which caused my tests to fail, which in turn enabled me to fix the situation.

So, a combination of testing + proper error checking leading to correct code ... is what is going on here.